modbus_sungrow.yaml

```diff
@@ -2181,7 +2181,7 @@ template:
           {% elif ((states('sensor.battery_forced_charge_discharge_cmd_raw') |int)  == 0x00CC) %}
             Stop (default)
           {% else %}
-            Unknown - should not see me!
+            Unknown - should not see me!  {{ (states('sensor.battery_forced_charge_discharge_cmd_raw') |int) }}
```

Comment:
Please add a small "code:" before the number to make it more clear for the user (for all 3 else-branches). When you make a PR, I will merge it into the main.


``` Unknown - should not see me!  {{ (states('sensor.battery_forced_charge_discharge_cmd_raw') |int) }}```

``` Unknown - should not see me! Code: {{ (states('sensor.battery_forced_charge_discharge_cmd_raw') |int) }}```